content/docs/standard/table-schema.md

```diff
@@ -266,6 +266,44 @@ If the value of the `foreignKey.reference.resource` property is an empty string
 Data consumer MUST support the `foreignKey.fields` and `foreignKey.reference.fields` properties in a form of a single string e.g. `"fields": "a"` which was a part of the `v1.0` of the specification.
 :::
 
+#### `name`
+
+A simple name or identifier for the schema (cf. [Data Package](https://datapackage.org/standard/data-package/#name)).
+
+#### `title`
+       
+A string providing a title or one sentence description for the schema.
+
+#### `description`
+       
+A description of the schema (cf. [Data Package](https://datapackage.org/standard/data-package/#description)).
+
+#### `homepage`
+       
+A URL for the home on the web that is related to the schema.
+
+#### `keywords`
+
+An array of string keywords to assist users searching for the schema in catalogs.
+
+#### `examples`
+
+A list of Data Resources that use and illustrate the schema.
+
+If present, it `MUST` be a non-empty array of objects. Each object is a [Data Resource](https://datapackage.org/standard/data-resource/) that `MUST` at least have the `name` and `path` property. The `path` must be a URL.
+
+#### `created`
+
+The datetime on which the schema was created (cf. [Data Package](https://datapackage.org/standard/data-package/#created)).
```

Comment:
```suggestion
The datetime on which the schema was created.
```

Reply:
Same here, I think this might inherit from Data Package.

Reply:
I agree